tests/quantization/mxfp4/test_mxfp4.py

```diff
@@ -348,7 +348,7 @@ def test_convert_moe_packed_tensors(self):
         self.assertEqual(result.dtype, torch.bfloat16)
 
     @require_triton(min_version="3.4.0")
-    @require_triton_kernels
+    @require_kernels
```

Comment:
Small suggestion: let's add `@require_kernels` and `@require_triton(min_version="3.4.0")` to the tests below in `Mxfp4ModelTest` that need this. The model path is needing to be updated also.